sympy/physics/control/control_plots.py

```diff
@@ -90,8 +94,8 @@ def pole_zero_numerical_data(system):
     >>> from sympy.physics.control.lti import TransferFunction
     >>> from sympy.physics.control.control_plots import pole_zero_numerical_data
     >>> tf1 = TransferFunction(s**2 + 1, s**4 + 4*s**3 + 6*s**2 + 5*s + 2, s)
-    >>> pole_zero_numerical_data(tf1)  # doctest: +SKIP
-    ([-1j, 1j], [-2.0, (-1.5-0.8660254j), (-0.5+0.8660254j)])
+    >>> pole_zero_numerical_data(tf1)
+    ([-1.0*I, 1.0*I], [-2.00000000000000, -1.00000000000000, -0.5 - 0.866025403784439*I, -0.5 + 0.866025403784439*I])
```

Comment:
This is a backwards incompatible change.

It may be best to make a new function that returns a list of SymPy types.

Reply:
```python
>>> Tuple([-1j, 1j], [-2.0, (-1.5-0.8660254j), (-0.5+0.8660254j)])
([-1.0*I, 1.0*I], [-2.0, -1.5 - 0.8660254*I, -0.5 + 0.8660254*I])
```